conanfile.txt

```diff
@@ -2,7 +2,7 @@
 ade/0.1.2a
 onetbb/[>=2021.2.1]
 pugixml/[>=1.10]
-protobuf/[>=3.20.3]
+protobuf/3.21.9
```

Comment:
not aligned with constraint files. Should we do that?

Reply:
no